training-react/login/src/app/components/App/components/Login/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Login from './layout';
+
+const LoginContainer = () => <Login />;
+
+const mapStateToProps = state => ({});
+
+const mapDispatchToProps = dispatch => ({});
+
+export default connect(
+  mapStateToProps,
```

Comment:
As this property is empty you couldn't send it to `connect`. Change this for: `export default connect()(LoginContainer)`